docs/source/guides/db_integration_with_featuretools_sql.ipynb

```diff
@@ -0,0 +1,125 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Connecting to your database instance \n",
+    "\n",
+    "Depending on your choice of DBMS, you may have to provide different pieces of information to the `DBConnector` object.\n",
+    "\n",
+    "If you want to connect to a `MySQL` instance, you should pass in the `system_name` as `mysql`. If you want to connect to a `Postgres` instance, you should pass in the `system_name` as `postgresql`.\n",
+    "\n",
+    "Here is an example call to the constructor of the object, connecting to a `Postgres` database:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```python \n",
+    "from featuretools_sql.connector import DBConnector\n",
+    "\n",
+    "connector_object = DBConnector(\n",
+    "    system_name=\"postgresql\",\n",
+    "    user=\"postgres\",\n",
+    "    host=\"localhost\",\n",
+    "    port=\"5432\",\n",
+    "    database=\"postgres\",\n",
+    "    schema=\"public\",\n",
+    ")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the choice of RDBMS does affect the required arguments -- for example, if you were connecting to a `MySQL` instance, you would not need a `schema` argument.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Converting to an EntitySet \n",
+    "\n",
+    "You can call the `get_entityset` method to instruct the `DBConnector` object to build an EntitySet. \n",
+    "\n",
+    "This method will loop through all the tables in the database and copy them into dataframes. Then it will populate the relationships data structure. It will finally pass those two arguments into the EntitySet constructor in Featuretools, and return the object."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```python \n",
+    "es = connector_object.get_entityset()\n",
+    "``` \n",
+    "\n",
+    "Optionally, you can pass in table names to the `select_only` parameter if you only want to include a subset of the tables in the database. \n",
+    "\n",
+    "```python \n",
+    "es = connector_object.get_entityset(select_only=[\"Products\", \"Transactions\"])\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This EntitySet object is ready to be passed into Featuretools's `DFS` algorithm! Read more about `DFS` [here]([https://featuretools.alteryx.com/en/stable/getting_started/afe.html). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Examining the EntitySet's member data \n",
+    "\n",
+    "You can examine the member data of the `DBConnector` object to ensure that it imported data correctly.\n",
+    "\n",
+    "To access the dataframes it imported, access the `.dataframes` attribute. To access the relationships data structure, access the `.relationships` attribute.\n"
```

Comment:
Show the user how to call DFS, and run automated feature generation.
We should also plot the EntitySet before we run DFS.

Reply:
Do we want something like this? https://featuretools.alteryx.com/en/stable/getting_started/afe.html#Running-DFS
If so, could we just link to it after showing how to plot the entityset?

Reply:
Yah it's okay to link for now. I think in the future we might have to get it working on that page.
Let's create an issue to track that.